DavidGonzalezZumaya/Gradle/src/main/java/JavaStaticInitializerBlock/JavaStaticInitializerBlock.java

```diff
@@ -0,0 +1,37 @@
+package JavaStaticInitializerBlock;
+
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class JavaStaticInitializerBlock {
+
+    static final String EXCEPTION_MESSAGE = "java.lang.Exception: Breadth and height must be positive";
+    static final String LOGGER_MESSAGE = "Static Initializer Log";
+
+    private static final Logger logger = Logger.getLogger(LOGGER_MESSAGE);
+    static Boolean flag = true;
+    static Scanner scanner = new Scanner(System.in);
+    static int breadth = scanner.nextInt();
+    static int height = scanner.nextInt();
+
+    static {
+        try {
+            if (breadth <= 0 || height <= 0) {
+                flag = false;
+                throw new Exception(EXCEPTION_MESSAGE);
+            }
+        } catch (Exception e) {
```

Comment:
Prefer specific Exceptions
https://dzone.com/articles/9-best-practices-to-handle-exceptions-in-java